packages/anndata/meta.yaml

```
@@ -0,0 +1,26 @@
+name: AnnData
+description: |
+  AnnData is a Python package for handling annotated data matrices in memory and on disk,
```

```suggestion
  anndata is a Python package for handling annotated data matrices in memory and on disk,
```

lowercase for the package?